runtime/compiler/optimizer/J9EstimateCodeSize.cpp

```diff
@@ -1294,7 +1297,13 @@ TR_J9EstimateCodeSize::realEstimateCodeSize(TR_CallTarget *calltarget, TR_CallSt
       return returnCleanup(ECS_VISITED_COUNT_THRESHOLD_EXCEEDED);
       }
 
-   if (_recursionDepth > MAX_ECS_RECURSION_DEPTH)
+
+   int32_t maxRecDepth =  MAX_ECS_RECURSION_DEPTH;
+   static const char *mrd = feGetEnv("TR_MaxRecursionDepth");
+   if (mrd)
+      maxRecDepth = atoi(mrd);
```

Comment:
It looks like the `atoi` will be performed every time `realEstimateCodeSize` is called if the environment variable has been set.  It might be better to make `maxRecDepth` into a `static const` as well, so that `atoi` can be called just once:

```
static const int32_t maxRecDepth = mrd ? atoi(mrd) : MAX_ECS_RECURSION_DEPTH;
```

Reply:
Addressed, thanks